apps/console/src/features/console-settings/components/console-administrators/console-administrators.tsx

```diff
@@ -50,15 +55,63 @@ const ConsoleAdministrators: FunctionComponent<ConsoleAdministratorsInterface> =
     const { isSubOrganization } = useGetCurrentOrganizationType();
 
     const [ activeAdministratorGroup, setActiveAdministratorGroup ] = useState("activeUsers");
+    const [ availableUserStores, setAvailableUserStores ] = useState<UserStoreDropdownItem[]>([]);
 
     const { t } = useTranslation();
 
+    const {
+        data: userStoreList,
+        isLoading: isUserStoreListFetchRequestLoading
+    } = useUserStores(null);
+
+    useEffect(() => {
+        if (userStoreList && !isUserStoreListFetchRequestLoading) {
+            const storeOptions: UserStoreDropdownItem[] = [
+                {
+                    key: -1,
+                    text: t("console:manage.features.users.userstores.userstoreOptions.primary"),
+                    value: "primary"
+                }
+            ];
+
+            let storeOption: UserStoreDropdownItem = {
+                key: null,
+                text: "",
+                value: ""
+            };
+
+            userStoreList?.forEach((store: UserStoreListItem, index: number) => {
+                if (store.name !== CONSUMER_USERSTORE) {
+                    getAUserStore(store.id).then((response: UserStorePostData) => {
+                        const isDisabled: boolean = response.properties.find(
+                            (property: UserStoreProperty) => property.name === "Disabled")?.value === "true";
+
+                        if (!isDisabled) {
+                            storeOption = {
+                                key: index,
+                                text: store.name,
+                                value: store.name
+                            };
+                            storeOptions.push(storeOption);
+                        }
+                    });
+                }
+            });
+
+            setAvailableUserStores(storeOptions);
+        }
+    }, [ userStoreList, isUserStoreListFetchRequestLoading ]);
+
     const renderSelectedAdministratorGroup = (): ReactElement => {
         switch (activeAdministratorGroup) {
             case "activeUsers":
-                return <AdministratorsList />;
+                return (<AdministratorsList
```

Comment:
```suggestion
                return (
                  <AdministratorsList
```

For readability, check other places.

Reply:
Fixed